--- parking_lot_core-0.7.0/Cargo.toml	2019-11-25T21:16:31+00:00
+++ parking_lot_core-0.7.0/Cargo.toml	2020-01-16T21:40:35.000863+00:00
@@ -28,7 +28,7 @@
 version = "0.1.5"
 
 [dependencies.petgraph]
-version = "0.4.5"
+version = "0.5"
 optional = true
 
 [dependencies.smallvec]
@@ -41,12 +41,5 @@
 [features]
 deadlock_detection = ["petgraph", "thread-id", "backtrace"]
 nightly = []
-[target."cfg(target_os = \"cloudabi\")".dependencies.cloudabi]
-version = "0.0.3"
-[target."cfg(target_os = \"redox\")".dependencies.redox_syscall]
-version = "0.1"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.55"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["winnt", "ntstatus", "minwindef", "winerror", "winbase", "errhandlingapi", "handleapi"]
